frontend/src/metabase/core/components/AccordionList/AccordionList.jsx

```diff
@@ -181,6 +180,12 @@ export default class AccordionList extends Component {
     }
   }
 
+  _getRootContainer() {
+    return this.isVirtualized()
+      ? this._list?.Grid?._scrollingContainer
```

Comment:
How did you come up with this value? How did you test it?

Reply:
I checked the source code of `react-virtualized` and found that `findDOMNode` was accessing this scrolling container. I will actually update this as 9.22.5 (the version I just updated to yesterday in github.com/metabase/metabase/pull/54645) now exposes the public `elementRef` prop (which is a lot better than using `_scrollingContainer` as that seems private)

Reply:
update: I had to revert the `elementRef` prop as it isn't really available in the 9.22.5 version despite the above diffs in their GitHub tag 😢 

https://github.com/metabase/metabase/pull/54650#discussion_r1983833493